src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -248,7 +248,7 @@ def validate_premium_registry(cli_ctx, registry_name, resource_group_name=None,
     """
     registry, resource_group_name = get_registry_by_name(cli_ctx, registry_name, resource_group_name)
 
-    if not registry.sku or registry.sku.name != SkuName.premium.value:
+    if not registry.sku or registry.sku.name != 'Premium':
```

Comment:
Much prefer using SDK values when possible.
Why this change? Does the SDK no longer document the skus available?